pkgs/servers/web-apps/phppgadmin/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub } :
+
+stdenv.mkDerivation rec {
+  name = "phppgadmin-${version}";
+  version = "7.13.0";
+
+  src = fetchFromGitHub {
+    owner  = "phppgadmin";
+    repo   = "phppgadmin";
+    rev    = "REL_7-13-0";
+    sha256 = "13bnnidzp0q7c6jz7vcfamfi884h8q654n3x3f2b2clfpng0cc0v";
+  };
+
+  phases = [
+    "unpackPhase"
+    "installPhase"
+  ];
```

Comment:
```suggestion
```

please disable phases you wish to not run instead.

Reply:
i have deleted the unpackPhase